Rules.md

```diff
@@ -1,16 +1,16 @@
 # Definitions
 
-* **Turn:** A single iteration of the game flow. There are a maximum of 10 turns in a game.
+* **Round:** A single iteration of the game flow. There are a maximum of 10 rounds in a game.
```

Comment:
This now clashes with the battle rounds. If we end up doing #206, this is not an issue. For now, I'd recommend at least renaming the battle rounds to something else (e.g. turns)

(or reverting this)